inception-workload-dynamic/src/main/java/de/tudarmstadt/ukp/inception/workload/dynamic/page/MonitoringPage.java

```diff
@@ -0,0 +1,456 @@
+import de.tudarmstadt.ukp.clarin.webanno.security.UserDao;
+import de.tudarmstadt.ukp.clarin.webanno.security.model.User;
+import de.tudarmstadt.ukp.clarin.webanno.ui.core.menu.MenuItemRegistry;
+import de.tudarmstadt.ukp.clarin.webanno.ui.core.page.ApplicationPageBase;
+import de.tudarmstadt.ukp.inception.workload.dynamic.support.DataProvider;
+import de.tudarmstadt.ukp.inception.workload.dynamic.support.Filter;
+import de.tudarmstadt.ukp.inception.workload.dynamic.support.ImagePanel;
+import de.tudarmstadt.ukp.inception.workload.dynamic.support.ModalPanel;
+
+
+@MountPath("/workload.html")
+public class MonitoringPage extends ApplicationPageBase
+{
+
+    private static final long serialVersionUID = 1180618893870240262L;
+
+    //Top Form
+    MonitoringForm workload;
+
+    //Dataprovider for table
+    private DataProvider dataProvider;
+
+    //Layout Panel
+    private DataTable table;
+
+    //Modal Window (popup panel on clicking the image in the metadata column)
+    private ModalWindow modalWindow;
+
+    //Default annotations textbox
+    private NumberTextField<Integer> DEFAULT_DOCUMENTS_NUMBER_TEXT_FIELD;
+
+    //FilterTextField and input
+    private TextField<String> filterTextfield;
+    private DropDownChoice<String> filterDropDown;
+
+    //The Icons
+    private static final ResourceReference META =
+        new PackageResourceReference(MonitoringPage.class, "book_open.png");
+
+    //Current Project
+    private IModel<Project> currentProject = new Model<>();
+
+    //List containing all documents
+    private IModel<List<SourceDocument>> documentList = new Model();
+
+
+    //Default annotations
+    public IModel<Integer> defaultAnnotations = new Model<>();
+
+
+    //SpringBeans
+    private @SpringBean UserDao userRepository;
+    private @SpringBean ProjectService projectService;
+    private @SpringBean MenuItemRegistry menuItemService;
+    private @SpringBean DocumentService documentService;
+
+
+    //Default constructor, no project selected (only when workload.html
+    // put directly in the browser without any parameters)
+    public MonitoringPage() {
+        super();
+
+        //top form initialize
+        workload = new MonitoringForm("workload");
+        workload.setOutputMarkupId(true);
+
+        //Required due to wicket
+        add(new ModalWindow("modalWindow"));
+        add(new NumberTextField("defaultDocumentsNumberTextField"));
+        add(new EmptyPanel("dataTable"));
+
+        //Error, user is returned to home page, nothing else to do
+        error("No Project selected, please enter the monitoring page only with a valid project reference");
+        setResponsePage(getApplication().getHomePage());
+
+    }
+
+
+
+    //Constructor with a project
+    public MonitoringPage(final PageParameters aPageParameters)
+    {
+        super(aPageParameters);
+
+        //top form initialize
+        workload = new MonitoringForm("workload");
```

Comment:
I think you don't need the class MonitoingForm at all (this conveniently also gets rid of the name-label problem ;) ). You don't use the label or the stored project model as far as I can see. `workload = new Form<>("workload");` should work.